lib/public/components/qcFlags/qcFlagsPagesButtons.js

```diff
@@ -36,15 +40,29 @@ export const qcFlagCreationPanelLink = ({ dataPassId, simulationPassId, runNumbe
         return;
     }
 
-    return frontLink(
-        content,
-        pageName,
-        { runNumber, dplDetectorId, ...options },
-        {
-            class: 'btn btn-primary',
-            title: 'Create a QC flag',
+    const disabledButton = h('button.btn.btn-primary', { disabled: true }, content);
+    return dplDetectorsProvider.physical$.getCurrent().match({
+        NotAsked: () => tooltip(disabledButton, `Cannot say whether you have permissions for detector with id ${dplDetectorId}`),
+        Loading: () => tooltip(disabledButton, 'Loading...'),
+        Failure: () => tooltip(disabledButton, `Cannot say whether you have permissions for detector with id ${dplDetectorId} because of error`),
```

Comment:
This seems a bit too familiar, I would rather put `Unable to determine permissions on detector ${detectorId}`

Reply:
applied